### PR TITLE
[INLONG-3883][Manager] Support create table of ClickHouse

### LINKED ISSUE #3883: [Feature] Add ClickHouse Dynamic Table Connector
### Description

[Feature] Add ClickHouse Dynamic Table Connector

### Use case

[Feature] Add ClickHouse Dynamic Table Connector

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lucaspeng12138**
### Title Name: [INLONG-3883][Manager]Support create table with clickhouse
Add create table to Manager Group.

fix #3883

### CODE REVIEW

**Comment by gong** [comment]
`git rebase master` before pull request.

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/ck/ClickHouseColumnInfo.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    private String name;
+    private String type;
+    private String desc;
+    private String defaultType;
+    private String defaultExpr;
+
+    private String compressionCode;
+
+    private String tTLExpr;
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/ck/ClickHouseColumnInfo.java:33]: Wouldn't it be better to replace `tTLExpr` with `ttlExpr`?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseSqlBuilder.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import org.apache.commons.lang3.StringUtils;
+import org.apache.inlong.manager.common.pojo.sink.ck.ClickHouseColumnInfo;
+import org.apache.inlong.manager.common.pojo.sink.ck.ClickHouseTableInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class ClickHouseSqlBuilder<T> {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseSqlBuilder.java:29]: Does this generics `T` needed?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseJdbcUtils.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+        if (conn == null) {
+            throw new Exception("get clickhouse connection failed, please contact administrator");
+        }
+
+        LOG.info("get clickhouse connection success, url={}", url);
+        return conn;
+    }
+
+    public static void executeSql(String sql, String url, String user, String password) throws Exception {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseJdbcUtils.java:71]: Please add some brief java docs for each static method.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/ck/ClickHouseColumnInfo.java (hunk 1 above)
Comment (lucaspeng12138) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/ck/ClickHouseColumnInfo.java]: Good idea, I will fix it.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseSqlBuilder.java (hunk 2 above)
Comment (lucaspeng12138) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseSqlBuilder.java]: You are right, I should fix it.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseJdbcUtils.java (hunk 3 above)
Comment (lucaspeng12138) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/ck/ClickHouseJdbcUtils.java:74]: OK, I have add description.

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**